theorems/T000143.md

```diff
@@ -0,0 +1,12 @@
+---
+uid: T000143
+if:
+  P000126: true
+then:
+  P000001: true
+refs:
+- mo: 405781
+  name: How do finite door spaces work?
+---
+
+If $x$ and $y$ are two topologically indistinguishable points, the singleton $\{x\}$ is neither open nor closed.
```

Comment:
Suggestion: either clarify that you're proceeding by contrapositive, or use this direct proof:

Given two distinct points $x,y$, either $\{x\}$ is an open neighborhood of $x$ not containing $y$, or $X\setminus\{x\}$ is an open neighborhood of $y$ not containing $x$; therefore the points are topologically distinct.

Reply:
(Github ate my backslashes it seems.)

Reply:
Yeah, we can clarify a bit.  Let me incorporate your change.